build/dockerfiles/rhel.Dockerfile

```diff
@@ -84,6 +93,7 @@ WORKDIR /var/www/html
 RUN mkdir -m 777 /var/www/html/devfiles
 COPY .htaccess README.md /var/www/html/
 COPY --from=builder /build/devfiles /var/www/html/devfiles
+COPY --from=dwtemplates /build/out /var/www/html/devfiles/v2/che-theia/
```

Comment:
my understanding was that the templates should go in the go sample folder if it's for go for example (and not che-theia) ?

Reply:
it was commited mistakenly, reverted the fixup